app/assets/stylesheets/style.css

```diff
@@ -0,0 +1,230 @@
+body {
+  background-color: rgba(240, 245, 245, 0.5);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style-type: none;
+}
+
+.group_card {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background: white;
+  margin: 24px;
+  border-radius: 24px;
+  padding: 12px;
+}
+
+.group_image {
+  height: 70px;
+  width: 70px;
+  border-radius: 12px;
+}
+
+.group_card_detail {
+  width: 70%;
+}
+
+.group_card_detail-inner {
+  display: flex;
+  justify-content: space-between;
+  height: 55px;
+  padding-top: 12px;
+}
+
+p.amount {
+  font-weight: bold;
+  padding: 8px;
+}
+
+.total_amount {
```

Comment:
* As required : 
* You should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
Colors.
Typography: font face, size and weight.
Layout: composition and space between elements.
* Kindly use the given fonts and colors
![image](https://github.com/rachaouldbabaali/budget-app/assets/52497246/e4951fbd-f4db-4599-ba3e-256eb4eecd20)
![image](https://github.com/rachaouldbabaali/budget-app/assets/52497246/3df6b57d-4669-4828-b081-48b6cf58e454)